### ISSUE TITLE
Docker image does not contain CA certificates

### ISSUE THREAD

**Issue by austinnichols101** [mattermost/mattermost-docker-preview#18]
To reproduce.

1.  Configure the GitLab / Mattermost `slash commands` integration
2.  Try a slash command (e.g. `/my-repo help`)

You will find the error message in the container console log file
```x509: failed to load system roots and no roots provided```

This is because the `ca-certificates` package has not been included in the container build so the mattermost process cannot verify the GitLab website via SSL (no certs to compare to).

As a workaround, I opened a shell to the running container and issued the following commands:

apt-get update
apt-get install ca-certificates

After restarting the **container instance**, the error/message should be eliminated from the console log.

**Comment by jasonblais** [comment]
We have an open community PR that adds a line to install ca-certificates. https://github.com/mattermost/mattermost-docker-preview/pull/17

Would this cover your use case here. If so, would you be interested to help with review?

**Comment by austinnichols101** [comment]
Yes, that should resolve the issue.  I will be happy to assist with the review.  Should we close this issue?

**Comment by jasonblais** [comment]
Thanks @austinnichols101, would love to have your help with review!

We can keep this issue open for now until the PR is merged, so others in the community are aware that this is a known issue.

**Comment by austinnichols101** [comment]
kk - I will test this out later today / tomorrow AM.

**Comment by austinnichols101** [comment]
Sorry - long day...tomorrow for sure.

**Comment by jasonblais** [comment]
Thanks @austinnichols101!

**Comment by jasonblais** [comment]
Many thanks @austinnichols101 for your interest in helping with the testing. If you find further issues, [you can file a GitHub issue in the /platform repo which is monitored more actively](https://www.mattermost.org/filing-issues/
